test/test_linalg.py

```diff
@@ -2360,7 +2360,7 @@ def gen_error_message(input_size, p, keepdim, dim=None):
                 expected = np.linalg.norm(xn, p, keepdims=keepdim)
                 msg = gen_error_message(x.size(), p, keepdim)
                 self.assertEqual(res.shape, expected.shape, msg=msg)
-                self.assertEqual(res, expected, msg=msg, rtol=1.3e-6, atol=3e-4)
+                self.assertEqual(res, expected, msg=msg, rtol=4e-6, atol=6e-4)
```

Comment:
How come we got a regression if we increased the number of iterations? If you don't know the reason I don't think you need to investigate, just curious.

Reply:
This is an example failing message on Windows only, does that give any hints? I honestly have no idea why it fails. There is no change to `norm`, and the `svd` default heuristic remains the same.

```
======================================================================
FAIL [0.016s]: test_norm_complex_old_cuda (__main__.TestLinalgCUDA)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\actions-runner\_work\pytorch\pytorch\build\win_tmp\build\torch\testing\_internal\common_device_type.py", line 376, in instantiated_test
    result = test(self, **param_kwargs)
  File "C:\actions-runner\_work\pytorch\pytorch\build\win_tmp\build\torch\testing\_internal\common_device_type.py", line 815, in dep_fn
    return fn(slf, *args, **kwargs)
  File "C:\actions-runner\_work\pytorch\pytorch\build\win_tmp\build\torch\testing\_internal\common_device_type.py", line 815, in dep_fn
    return fn(slf, *args, **kwargs)
  File "test_linalg.py", line 2363, in test_norm_complex_old
    self.assertEqual(res, expected, msg=msg, rtol=1.3e-6, atol=3e-4)
  File "C:\actions-runner\_work\pytorch\pytorch\build\win_tmp\build\torch\testing\_internal\common_utils.py", line 2170, in assertEqual
    assert_equal(
  File "C:\actions-runner\_work\pytorch\pytorch\build\win_tmp\build\torch\testing\_comparison.py", line 1074, in assert_equal
    raise error_metas[0].to_error()
AssertionError: complex norm failed for input size torch.Size([25, 25]), p=nuc, keepdim=True, dim=None
Tensor-likes are not close!

Mismatched elements: 1 / 1 (100.0%)
Greatest absolute difference: 0.0005035400390625 at index (0, 0) (up to 0.0003 allowed)
Greatest relative difference: 3.323382990382029e-06 at index (0, 0) (up to 1.3e-06 allowed)

```